doc/tools/make_rst.py

```diff
@@ -1307,7 +1306,7 @@ def format_text_block(
                 if cmd == "gdscript":
                     if not inside_code_tabs:
                         print_error(
-                            "{}.xml: GDScript code block is used outside of [codeblocks] in {}.".format(
+                            "{}.xml: GDScript code block is used outside of [codeblocks] in {} / {}.".format(
                                 state.current_class, cmd, context_name
```

Comment:
In that case, just remove `cmd`, as that's the proper fix here and below.

Reply:
Welp, I copy-pasted the f-strings from other PR in the meantime. But that feels like a better thing, since it will make the conflict resolution easier for whoever will have to deal with it.

Reply:
#64545 is merged, which solves how to deal with this here. Rebased.